test-suite/output/bug_7443.out

```diff
@@ -1,7 +1,13 @@
 Literal 1
      : Type
+File "./output/bug_7443.v", line 21, characters 2-46:
+Warning: Notation "[ _ ]" was already used in scope foo_scope.
+[notation-overridden,parsing]
```

Comment:
This warning seems reasonable as we are overriding the notation, does anyone disagree?

Reply:
Looks expected, indeed. So, wasn't it issued because of the `ltac:(...)`?

Reply:
The notation comparison saw `(Literal (t:=reify_type_of x) x)` ie `(@Literal (reify_type_of x) x)` as `(@Literal _ x)` so equal to the second notation `(Literal x)`
Now genarg and hole don't compare equal so it sees a difference.